tests/unit/test_sde.py

```diff
@@ -151,7 +155,7 @@ def test_KL_sde(setup):
     Test the KL_sde function which calculates the KL divergence between two SDEs with same diffusion using
     Girsanov theorem.
 
```

Comment:
The test builds two different ssms, q and p, and compares two ways to evaluate KL[q|p]